website/docs/docs/collaborate/git/resolve-merge-conflicts.md

```diff
@@ -1,70 +1,72 @@
 ---
-title: "Resolve merge conflicts"
+title: "Merge conflicts in dbt Cloud"
```

Comment:
```suggestion

```

Reply:
sugg: 

- the first paragraph refers to dbt Cloud / IDE. maybe you can refer to it in the first sentence instead of the last. 
- breaks consistency in the leftbar: 
![Screen Shot 2022-11-23 at 10 36 43 AM](https://user-images.githubusercontent.com/107218380/203623096-a59b18c0-f708-49be-b999-68ffee7a1aa7.png)
- - maybe just "Merge conflicts" instead? if you do, consider changing the filename from `resolve-merge-conflicts.md` to `merge-conflicts.md`

Reply:
ah ok good point!
ok will do

Reply:
awesome, all set for you @nghi-ly to review when you're back next week.